app/controllers/course/experience_points_records_controller.rb

```diff
@@ -9,7 +9,7 @@ class Course::ExperiencePointsRecordsController < Course::ComponentController
 
   def index # :nodoc:
     @experience_points_records =
-      @experience_points_records.active.includes(:updater).order(updated_at: :desc)
+      @experience_points_records.active.includes(:updater, :actable).order(updated_at: :desc)
```

Comment:
The N+1 is triggered because we need `submission.assessment.name`? Or we need quite a few objects from the assessment object?